xml/hwbp_registers.xml

```diff
@@ -260,6 +260,10 @@
         \unspecified\ whether a load trigger with \FcsrMcontrolSelect=1 will
         match.  Whether store triggers with \FcsrMcontrolSelect=1 match on
         AMOs is \unspecified.
+
+        If \RcsrTextraThirtytwo or \RcsrTextraSixtyfour are implemented, this
```

Comment:
How about "are implemented for this trigger" on all of these?

Reply:
Good idea. Done.